2nd-gen/alerts-to-discord/functions/index.js

```diff
@@ -86,7 +91,49 @@ id: \`${id}\`
     // [END v2SendToDiscord]
   } catch (error) {
     logger.error(
-        `Unable to post fatal Crashlytics alert ${id} to Discord`,
+        `Unable to post fatal Crashlytics alert ${id} for ${appId} to Discord`,
+        error,
+    );
+  }
+});
+
+/**
+ * function triggered by AppDistribution that publishes a message
+ * to Discord whenever a new iOS tester device is registered.
```

Comment:
Add a link to product docs: Firebase Alerts triggers (https://firebase.google.com/docs/functions/beta/alert-events) and Register additional iOS devices (https://firebase.google.com/docs/app-distribution/register-additional-devices#trigger-function-new-udids). Note that Eventarc updates to these docs will be published on Oct. 18, so you might want to wait until then to add these links here.

Reply:
Yes, let's do that in a follow-up PR when those links exist.